src/gluonts/model/statsforecast.py

```diff
@@ -0,0 +1,329 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+import numpy as np
+
+from statsforecast.models import (
+    ADIDA,
+    AutoARIMA,
+    AutoCES,
+    CrostonClassic,
+    CrostonOptimized,
+    CrostonSBA,
+    ETS,
+    IMAPA,
+    TSB,
+)
+
+from gluonts.core.component import validated
+from gluonts.dataset import DataEntry
+from gluonts.dataset.util import forecast_start
+from gluonts.model.predictor import RepresentablePredictor
+from gluonts.model.forecast import QuantileForecast
+
+
+@dataclass
+class ModelConfig:
+    quantile_levels: Optional[List[float]] = None
+    forecast_keys: List[str] = field(init=False)
+    statsforecast_keys: List[str] = field(init=False)
+    intervals: Optional[List[int]] = field(init=False)
+
+    def __post_init__(self):
+        self.forecast_keys = ["mean"]
+        self.statsforecast_keys = ["mean"]
+        if self.quantile_levels is None:
+            self.intervals = None
+            return
+
+        intervals = set()
+
+        for quantile_level in self.quantile_levels:
+            interval = round(
+                200 * (max(quantile_level, 1 - quantile_level) - 0.5)
+            )
+            intervals.add(interval)
+            side = "hi" if quantile_level > 0.5 else "lo"
+            self.forecast_keys.append(str(quantile_level))
+            self.statsforecast_keys.append(f"{side}-{interval}")
+
+        self.intervals = sorted(list(intervals))
```

Comment:
```suggestion
        self.intervals = sorted(intervals)
```

Sorted returns a list already.